### ISSUE TITLE
[Feature Request] Reroute node widget with on/off switch and reroute node widget with patch selector 

### ISSUE THREAD

**Issue by fabiobtex** [comfyanonymous/ComfyUI#1051]
Reroute node widget with on/off switch and reroute node widget with patch selector 

-A reroute node (usually for image) that allows to turn off or on that part of workflow just moving a widget like switch button, exemple:
Turn on off if the image should go to face restore or not using switch widget. 

-A reroute node (usually for image) that allows you to choose the path of workflow the image should go, exemple:
Two completely different upscale processes, you use the switch to choose between PATH A and PATH B using the widget switch

**Comment by n00mkrad** [comment]
Yes, we badly need some sort of if/else conditions.

RIght now it's very tedious to switch between paths (e.g. upscaling on/off, highres fix on/off).

**Comment by Nikerino** [comment]
As a temporary solution (that won't apply to all cases) - I use https://github.com/LucianoCirino/efficiency-nodes-comfyui
This custom node suite has samplers that let you specify whether they should "Sample" or "Hold" - meaning they won't sample and will just pass on to the next part of the workflow. You can achieve a toggleable HiRez workflow this way as well.

**Comment by fabiobtex** [comment]
> As a temporary solution (that won't apply to all cases) - I use https://github.com/LucianoCirino/efficiency-nodes-comfyui This custom node suite has samplers that let you specify whether they should "Sample" or "Hold" - meaning they won't sample and will just pass on to the next part of the workflow. You can achieve a toggleable HiRez workflow this way as well.

Ty for the suggestion

**Comment by fabiobtex** [comment]
> Yes, we badly need some sort of if/else conditions.
> 
> RIght now it's very tedious to switch between paths (e.g. upscaling on/off, highres fix on/off).

Yes. I want to make complex full of options workflow more user more user friendly. I don't mind to turn "never" some key nodes and connect and disconnect noodles manually buy i can't share my workflow unless is for someone that is really experienced.

Aren't you the NMKD guy?
I started SD with NMKD gui in November. Very nice, easy to use. Ty for you work
